cmd/dump.go

```diff
@@ -75,6 +76,10 @@ func doDump(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	sort.Slice(lines, func(i, j int) bool {
+		return lines[i] < lines[j]
+	})
```

Comment:
We can write this more simpler way.

```suggestion
	sort.Strings(lines)
```

https://golang.org/pkg/sort/#Strings